src/frontend/components/Felleskomponenter/LenkeMedIkon/LenkeMedIkon.test.tsx

```diff
@@ -1,16 +0,0 @@
-import React from 'react';
-
-import { render, screen } from '@testing-library/react';
-
-import LenkeMedIkon from '../LenkeMedIkon/LenkeMedIkon';
-
-test('Kan rendre LenkeMedIkon', () => {
-    render(
-        <LenkeMedIkon
```

Comment:
Slettet kode som ikke var brukt, gikk selv i fella og brukte litt tid på dette så da tenkte jeg det var bedre å slette det for nestemann